python_ta/cfg/visitor.py

```diff
@@ -128,6 +129,20 @@ def visit_functiondef(self, func: nodes.FunctionDef) -> None:
         self._current_cfg.start.add_statement(func.args)
         func.cfg_block = self._current_cfg.start
 
+        if z3 is not None:
+            # Parse types
+            expr = ExprWrapper(func)
+            z3_vars_list = expr.parse_function_def(func)
+            # Confirm that type list is given
+            if isinstance(z3_vars_list, list):
```

Comment:
the return type of the `parse_function_def` method is a list, so this check is redundant